lib/FileUploadDataSource.ts

```diff
@@ -143,6 +144,10 @@ export default class FileUploadDataSource extends RemoteGraphQLDataSource {
         http: httpResponse,
       };
 
+      if (typeof this.didReceiveResponse === 'function') {
+        return this.didReceiveResponse({ context, request, response });
```

Comment:
@cabelitos minor, but here it just expects these 3? or should we pass along something like `args` or some other existing object?

Reply:
just these 3